CLI/commands/transfer_manager.js

```diff
@@ -396,8 +396,8 @@ async function generalTransferManager() {
       let now = Math.floor(Date.now() / 1000);
       let fromTime = readlineSync.questionInt(`Enter the time (Unix Epoch time) when the sale lockup period ends and the investor can freely sell his tokens (now = ${now}): `, { defaultInput: now });
       let toTime = readlineSync.questionInt(`Enter the time (Unix Epoch time) when the purchase lockup period ends and the investor can freely purchase tokens from others (now = ${now}): `, { defaultInput: now });
-      let oneHourFromNow = Math.floor(Date.now() / 1000 + 3600);
-      let expiryTime = readlineSync.questionInt(`Enter the time till investors KYC will be validated (after that investor need to do re - KYC) (1 hour from now = ${oneHourFromNow}): `, { defaultInput: oneHourFromNow });
+      let oneYearFromNow = Math.floor(Date.now() / 1000 + (60 * 60 * 24 * 365));
+      let expiryTime = readlineSync.questionInt(`Enter the time until the investors KYC will be valid (after this time expires, the investor must re-do KYC) (1 year from now = ${oneYearFromNow}): `, { defaultInput: oneYearFromNow });
       let canBuyFromSTO = readlineSync.keyInYNStrict('Is the investor a restricted investor?');
```

Comment:
This question needs to be changed as it causes confusion. It had me scratching my head a few weeks ago when I was getting invalid transfers. Similarly it needs to be changed in the Token manager.
```suggestion
       let canBuyFromSTO = readlineSync.keyInYNStrict('Can the investor buy from security token offerings?');
```

Reply:
Thanks @F-OBrien. I always try to keep these texts as similar to comments in contracts bun in this case this is much descriptive than the original.

Reply:
No Problem. This also needs to be changed in the Token Manager. There are a few other things that can be fixed in the CLI's also. Take a look at some of the commits #541 I made against Dev-3.0.0  as some of the changes should be applied to Dev-2.1.0 also. (I see Adam has requested you review it earlier today)

Reply:
Yes, I checked them. A couple of them were already included in this PR. Some others are changes regarding 3.0.0 version of contracts, so I can't include them here.
Once dev-2.1.0 is merged into dev-3.0.0, I'll approve and merge your PR too.